cypress/support/pages/admin_console/Masthead.ts

```diff
@@ -50,9 +50,11 @@ export default class Masthead {
     cy.get("#manage-account").click();
   }
 
-  checkNotificationMessage(message: string) {
+  checkNotificationMessage(message: string, closeNotification?: boolean) {
     cy.contains(message).should("exist");
-
+    if (closeNotification) {
+      cy.get(".pf-c-alert__action").click();
```

Comment:
Should this not be only the notification that matches the message? This way it might close another notification. Something like:
```suggestion
      cy.contains(message).closest(".pf-c-alert__action").click();
```

Reply:
i did go down this exact road of matching the exact message but it was failing... i can take another look

Reply:
Hmmm, yes this seems to fail quite hard. I wonder why, perhaps it's a sibling element?

Reply:
not sure, but i did do a bunch of testing with this PR's implementation and it works flawlessly, so i went with it. and there aren't arbitrary waits anymore, so i do think it's a viable solution for the test suite.